README.md

```diff
@@ -67,33 +127,6 @@ Base template has been designed and developed by Themefisher. For a preview, che
 ### Code and Contribution
 The code has been restructured, unnecessary code has been removed, comments were added wherever required, proper indentation was provided, and simple names were used which makes it a contributor-friendly codebase. So, hope on and contribute now!
 
-## How to run the code
-### Getting the code
-
-1. Fork this Repository.
-
-2. Clone your forked copy of the project.
-```
-git clone https://github.com/<your_user_name>/www-sugarlabs.git
-```
-3. Navigate to the project directory 📁.
-```
-cd www-sugarlabs
-```
-
-### Installation
-Install Ruby and the Gem package manager (pre-installed in Mac OS X) and install `jekyll` and its related packages available in the Github hosted version by running, from this directory:
-
-    bundle install;
-### Running
-Jekyll can run a local web server that rebuilds each time you save a page while editing (Execute the below command from root directory):
-
-    bundle exec jekyll serve --incremental;
-
-Then open [http://localhost:4000](http://localhost:4000)
-### Note
-If some changes are not visible (after saving a page while editing), please restart the Jekyll local web server and reload [http://localhost:4000](http://localhost:4000).
-
```

Comment:
In the end, all you have done is removed the policies you disagree with, and move things around in the file.